syft/frameworks/torch/fl/dataset.py

```diff
@@ -315,6 +325,8 @@ def workers(self):
 
         return list(self.datasets.keys())
 
+    locations = workers
```

Comment:
Q: Is this used?

Reply:
It's an alias which is more intuitive, we always have .location(s), except for this one